Formula/a/apko.rb

```diff
@@ -53,11 +53,9 @@ def install
       environment:
         PATH: /usr/sbin:/sbin:/usr/bin:/bin
 
-      # no keys found for arch loongarch64 and releases [edge],
-      # so constraint the archs to x86_64 and aarch64
+      # only key found for arch riscv64 [edge],
```

Comment:
actually this is a flaky issue

![image](https://github.com/user-attachments/assets/611dda67-8fd4-4bec-b4f8-5bd3493a80f3)

![image](https://github.com/user-attachments/assets/e124744f-b53b-48d8-81a8-be8c63b5e1ec)